### PR TITLE
War-crime  Removal With Red Reduction 

### PR DESCRIPTION
**Pull request by Hypertripp**
## About the PR 
This PR changes all red crosses in to white ones, some with red outlines and a few here and there another relevant color

Thanks to Arthanas#1530 for both the idea and much of the work!
## Why's this needed? 
Article 38(1) of the 1977 (Geneva convention) Additional Protocol I provides: “It is prohibited to make improper use of the distinctive emblem of the red cross, red crescent or red lion and sun."

## Changelog

```changelog
(u)Hypertripp
(*)Made red crosses not red 
```

### CODE REVIEW

**Comment by Eagletanker** [comment]
could you post screenshots of em?

**Comment by Gannets** [comment]
I'm gonna have to see a few screenshots of this.

**Comment by alexeric123** [comment]
https://github.com/goonstation/goonstation/pull/6554/checks?check_run_id=4087006639

**Comment by Hypertripp** [comment]
> I'm gonna have to see a few screenshots of this.

there are a lot of little things changed your going to have to specify what exactly you want to see.

**Review by ZeWaka — CHANGES_REQUESTED** [review]
side views of computerpanel.dmi and computer.dmi and computer_screens.dmi bad
icons/obj/items/vendcart.dmi/medical_public looks wack
icons/obj/items/cyborg_parts/modules.dmi i can't tell what it is anymore
icons/obj/clothing/overcoats/item_suit_gimmick.dmi & icons/mob/overcoats/worn_suit_gimmick.dmi/guardscoat you improperly merged this and regressed these icons
icons/mob/monkey_clothes.dmi you added some wack stuff and regressed
icons/obj/chemical.dmi you can't just remove the cross
icons/obj/items/pda.dmi/cart_med gross pink
icons/mob/overcoats/worn_suit.dmi regression
icons/obj/bots/medbots.dmi a lot of these look really wonky
icons/obj/items/storage.dmi weird addition


tl;dr:
you should be using the icon merge driver and not have dream maker open when pulling from master
this PR also could've been split up a bit more

**Comment by FlamingLily** [comment]
Why not just make the crosses green? Instead of white with a red outline? I'd argue a green cross is more recognizable and should look better

**Comment by DimWhat** [comment]
I am generally against this PR.This has came up a few times in discussions it I've always been for changing up medbay to stop being a warcrime... however these crosses seem pretty hard to read when you make them white with a red outline. As well as several of the changes you've made make the sprites look worse, harm the item's identity, are completely unnecessary, or don't even follow the design rules you've made with this change.
Looking Worse:
image
Massacred among many others, this just looks bad.

Harming the identity: 
image
This hurts patches so very bad. They've changed from being a distinguishably _medical_ patch to a generic patch.

Completely Unnecessary:
image
As you said, **red** crosses are the problem. Anything that is not a **red** cross should not be considered an issue. The flooring among many things do not have a red cross. These do not need any changes as they are not even problematic to begin with.

Following your own rules:
image
Here's some recoluring of the cross, yet everything else was turned to be a white cross on a red background.

While I do understand it's a good thing to not break the Geneva Convention, I don't believe this is the correct approach in doing that. Being hard to read if something is a cross or tied to medbay is a HUGE problem and will be incredibly present if this PR goes through as is. There are however alternate solutions. One idea I've fiddled with occasionally was to instead just rebrand medical to be blue instead of red. There's already blue medical stuff laying around. Medbay itself and the medical channel are coloured blue.  I find that to be a preferable alternative to this.

Please note my opinion isn't a be all end all solution, I'm not even certain it's what people want.

**Comment by Hypertripp** [comment]
> Why not just make the crosses green? Instead of white with a red outline? I'd argue a green cross is more recognizable and should look better

it was sort of agreed on earlier that white was where we are going and green is genetics' color but in light of all this I think its not a half bad idea

**Comment by Zonespace27** [comment]
Personally not a fan of this look at all tbh

**Comment by Eagletanker** [comment]
Just make it a Red N. Nanohealth, a subsidiary of Nanotrasen
